backend/app/api/endpoints/books.py

```
@@ -2791,3 +3050,102 @@ async def generate_chapter_draft(
         raise HTTPException(
             status_code=500, detail=f"Error generating draft: {str(e)}"
         )
+
+@router.post(
+    "/{book_id}/chapters/{chapter_id}/questions/responses/batch",
+    response_model=Dict[str, Any]
+)
+async def save_question_responses_batch_endpoint(
+    book_id: str,
+    chapter_id: str,
+    responses: List[Dict[str, Any]] = Body(...),
```

Ruff (0.14.10)

Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)